README.md

```diff
@@ -19,12 +19,28 @@ Include the package locally in your repository.
 
 `npm install dotenv-webpack --save`
 
+### Description
+
+`dotenv-webpack` wraps `dotenv` and `Webpack.DefinePlugin`. As such, it overwrites existing any existing `DefinePlugin` configurations. Also, like `DefinePlugin`, it does a text replace in the resulting bundle for any instances of `process.env`.
+
+Also, be aware that all information in your `.env` file will be included in the resulting bundle. Please do not share any secret information in your client bundle. Instead, make a separate `.client.env` file.
```

Comment:
This can be removed due to #42 so you can remove this paragraph. If you would rather we can simply state:

```
Your `.env` files can include sensative information. Because of this,`dotenv-webpack` will only include defined environment variables in the final bundle.
```